specification/storage/Storage.Management/StorageAccountMigration.tsp

```diff
@@ -0,0 +1,48 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+import "./StorageAccount.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.Storage;
+/**
+ * The parameters or status associated with an ongoing or enqueued storage account migration in order to update its current SKU or region.
+ */
+@parentResource(StorageAccount)
+model StorageAccountMigration
```

Comment:
why remove id, name and type?
<img width="875" height="380" alt="image" src="https://github.com/user-attachments/assets/e1d91ae2-fb11-4d26-b0f4-155957aab8c6" />